modules/sdk/gradle-plugins-whip/README.markdown

```diff
@@ -37,8 +37,9 @@ repositories {
 }
 ```
 
-By default, Whip is automatically applied to all the tasks of type [`Test`](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html).
-If a task has Whip applied and it is [enabled](#enabled), then Whip is configured as a Java Agent.
+By default, Whip is automatically applied to all tasks of type [`Test`](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html).
+If a task has Whip applied and is [enabled](#enabled), then Whip is configured
+as a Java Agent.
```

Comment:
@codyhoag , @Ithildir - Is it the task or Whip that must be enabled for Whip to be configured as a Java Agent?

My guess is that Whip must be enabled, but I wanted to check. Thanks.